variables.yaml

```diff
@@ -74,25 +74,25 @@ netcdf_variables:
     coordinates:  time depth latitude longitude
     instrument:    instrument_ctd
     valid_min:    "0."
-    valid_max:    "10."
+    valid_max:    "85."
     observation_type: "measured"
-    accuracy:      "0.0003"
-    precision:     "0.0001"
-    resolution:     "0.00002"
+    accuracy:      "0.003"
+    precision:     "0.001" #?
```

Comment:
Thanks for fixing this! I think I copied it from the GPCTD originally, hence the incorrect values. Can drop the precision field if there's no manufacturer supplied calue for it. These are all "nice to have" extras. The  essentials are:

    source:      
    long_name:    
    standard_name: 
    units:   
    coordinates:  
    
    Which you have